src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -1001,6 +1001,24 @@ public void TestConversionOperatorDeclaration()
                 "public static implicit operator global::System.Decimal(global::System.Byte value)\r\n{\r\n}");
         }
 
+        [Fact, WorkItem(77101, "https://github.com/dotnet/roslyn/issues/77101")]
+        public void TestExplicitOperatorFromSymbol()
+        {
+            var compilation = CSharpCompilation.Create("Test", [SyntaxFactory.ParseSyntaxTree("""
+                public class C
+                {
+                    public static explicit operator int(C c) => 0;
+                }
+                """)]);
+
+            var c = compilation.GetTypeByMetadataName("C");
+            var op = c.GetMembers().OfType<IMethodSymbol>().Where(m => m.MethodKind == MethodKind.Conversion).Single();
```

Comment:
This will need a filter to find the checked keyword.

Reply:
Naw, since I only added a single operator it's fine.